packages/web-components/@sfx/base/src/utils.ts

```diff
@@ -0,0 +1,27 @@
+/* istanbul ignore file */
+
+export function makeSlot(name?: string) {
+  const slot = document.createElement('slot');
+  if (name) {
+    slot.name = name;
+  }
+  return slot;
+}
+
+export function createChildrenObserver(root: any) {
```

Comment:
The type of `root` is known; it's probably `Node`.

```suggestion
export function createChildrenObserver(root: Node) {
```